package.json

```diff
@@ -78,6 +78,7 @@
     "gulp-sass": "3.1.0",
     "gulp-sourcemaps": "2.4.1",
     "gulp-stylelint": "3.9.0",
+    "gulp-theo": "^1.0.0-alpha.4",
```

Comment:
@stiaan I don't think I added anything to the `package.json` just to the script. I can't see anything we use dependant on it on npm: https://www.npmjs.com/package/gulp-theo 

Last thing to check would be uninstall it and see if you can still build?